cognite/neat/_session/_subset.py

```diff
@@ -0,0 +1,64 @@
+import warnings
+
+from cognite.neat._alpha import AlphaFlags
+from cognite.neat._issues._base import IssueList
+from cognite.neat._rules.models.entities._single_value import ClassEntity, ViewEntity
+from cognite.neat._rules.transformers import SubsetDMSRules, SubsetInformationRules
+
+from ._state import SessionState
+from .exceptions import NeatSessionError, session_class_wrapper
+
+try:
+    from rich import print
+except ImportError:
+    ...
+
+
+@session_class_wrapper
+class SubsetAPI:
```

Comment:
I am wondering if there is a simpler name, subset is a technical term that might not encourage our less technical users.